markdownreveal/commands.py

```diff
@@ -54,15 +54,22 @@ def cli():
     default=8123,
     help='Listen on port (default: 8123).',
 )
+@click.option(
+  '-n',
+  '--no-warmup',
+  is_flag=True,
+  help='Do not display the warmup slide, even if it exists in the'
+       ' style folder (default: false).'
+)
 def show(markdown_file: Path, host: str = 'localhost', port: int = 8123):
     """
     Visualize your presentation (default).
     """
     markdown_file = Path(markdown_file)
     config = load_config()
-
+    
```

Comment:
Mind removing these spaces? You can use `black` to format all the code for you. :blush: